package.json

```diff
@@ -33,11 +33,13 @@
     "precommit:backend": "cd backend && npm run precommit",
     "precommit:worker": "cd worker && npm run precommit",
     "precommit:shared": "cd shared && npm run precommit",
-    "precommit:root": "lint-staged && sh check-env-format.sh",
+    "precommit:root": "lint-staged && npm run detect-secrets:precommit && sh check-env-format.sh",
```

Comment:
Could we add a pre-commit check to see if `detect-secrets` has been installed on the machine, and if it hasn't, throw an error? 

This is to prevent a new engineer on the team who hasn't run the `pip install detect-secrets==1.2.0` command from accidentally committing secrets

Reply:
Oh if `detect-secrets` has not been installed, it will throw error as well (i.e. `command not found`) so we don't have to worry